README.md

```diff
@@ -119,15 +162,20 @@ The `debugWrapInScript` variable -- defaulting to `false` -- turns on the `set +
   
 The `baseEnvironmentScript` is sourced with relaxed values for `set`, i.e. with `set +ue`, because often files like `/etc/profile` are not under the control of the person running the workflow. Conversely, changes to the `set` options in the `baseEnvironmentScript` are not inherited by subsequent code in the  `wrapInScript.sh`.
```

Comment:
there is a doubled whitespace at the end of the sentence: `code in the  \`wrapInScript.sh\`.`

Reply:
would you like to keep it?